beangulp/importers/csv.py

```diff
@@ -73,6 +73,9 @@ class Col(enum.Enum):
     # account.
     CATEGORY = '[CATEGORY]'
 
+    # A column that indicates the amount currency for the current row which may be different to the base currency.
```

Comment:
This line is to long. Can you please wrap the comment a 78 columns or there about?